src/Type/ObjectType/Avatar.php

```diff
@@ -14,6 +16,7 @@ public static function register_type() {
 			'Avatar',
 			[
 				'description' => __( 'Avatars are profile images for users. WordPress by default uses the Gravatar service to host and fetch avatars from.', 'wp-graphql' ),
+				'model'       => AvatarModel::class,
```

Comment:
Possibly beyond the scope of this PR, but maybe we should abstract this to a filterable getter() instead of hardcoding the model type?

I know this PR is just focused on displaying them in query analyzer, but the logical next step would be to have `TypeRegistry->register_type()` actually use this define the model for the object. 

cc #2466

Reply:
Ya, we can definitely iterate on this. Right now the intent of this is to be able to identify the relationship between a node that was resolved and the Types that were asked for. 

Sometimes Models are loaded, but not returned. 

For example, a query for `{ posts { nodes { id, title } } }` will load the User Model for the author(s) of the posts (to determine ownership, etc), but not return those models because the query didn't ask for them. 

So, if each Type identifies the Model that's used to resolve its node, as is done here, then we can identify the intersection between Models/Nodes that were loaded and those that were asked for in the query. 

Ultimately we can use this Model definition elsewhere in resolvers/loaders, etc.